nominees/openrims.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "OpenRIMS",
+  "aliases": [
+    "Pharmadex"
+  ],
+  "description": "Digital Suite for Regulatory Authorities",
```

Comment:
Please visit the [website](https://openrims.org) and update the description field with a more detailed information about the project. Thanks.

Reply:
Fixed in commit 5bb862e